cirq/sim/density_matrix_utils_test.py

```diff
@@ -26,6 +26,38 @@ def assert_valid_density_matrix(matrix, num_qubits=1):
                                      dtype=matrix.dtype), matrix)
 
 
+def test_von_neumann_entropy():
+    # 1x1 matrix
+    assert cirq.von_neumann_entropy(np.array([[1]])) == 0
+    # An EPR pair state (|00> + |11>)(<00| + <11|)
+    assert cirq.von_neumann_entropy(
+        np.array([1, 0, 0, 1] * np.array([[1], [0], [0], [1]]))) == 0
+    # Maximally mixed state
+    assert cirq.von_neumann_entropy([[0.5, 0], [0, 0.5]]) == 1
+    # 3x3 state
+    assert np.isclose(
+        cirq.von_neumann_entropy(
+            np.array(
+                np.array([[0.5, 0.5j, 1], [-0.5j, 0.5, 0], [0.7, 0.4, 0.6]]))),
+        1.9, 1e01)
+    # 4X4 state
+    assert np.isclose(
+        cirq.von_neumann_entropy(
+            np.array([[0.5, 0.5j, 1, 3], [-0.5j, 0.5, 0, 4], [0.7, 0.4, 0.6, 5],
```

Comment:
Use `# yapf: disable` and `# yapf: enable` to keep these matrices lined up.

Reply:
Done.